src/features/uiColumnFilter/templates/filters/numberColumnFilter.html

```diff
@@ -0,0 +1,42 @@
+<select 
+    ng-model="col.colDef.columnFilter.operators[0]" 
+    class="form-control">
```

Comment:
The bootstrap that we include is namespaced. This will show up differently depending upon whether someone has bootstrap installed.